public/89889688147bd7575d6327160d64e760.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Please ignore it in general. It must have been from a webpack build. I have deleted it and rebuilt. File was not recreated.